frontend/views/utils/vueComponentStub.js.flow

```diff
@@ -0,0 +1,3 @@
+const x: Object = {}
+export default x
```

Comment:
@snowteamer Is it possible to mess with either this file or with the `.flowconfig` to prevent having to add `: Object)` everywhere?

For example, did you try adding that here instead?

```js
export default (x: Object)
```

Or something to that effect?

Reply:
I don't think so, as this file only acts as a stub for import statements, not for exports. Upon encountering a `*.vue` import statement, Flow will use this stub instead of the Vue file (which might even not exist) and assume the resulting type must be `Object`.

Therefore the config file only allows to mess with imports, not exports.

Flow always requires explicit typing at so-called module boundaries (i.e. module exports).

Quoted from the [Flow blog](https://medium.com/flow-type/asking-for-required-annotations-64d4f9c1edf8):

> #### Why are annotations required?
>
> Flow has very few limitations on its type inference, but there is one big one: input positions reachable from exports must be annotated. An inferred type in such a position is insufficient, as it would not account for a value that might “flow in” from a dependent module. Indeed, these types are often never constrained by the defining module.